gamemode/modules/base/cl_entityvars.lua

```diff
@@ -1,16 +1,20 @@
 local DarkRPVars = {}
 
 --[[---------------------------------------------------------------------------
-interface"someString"
```

Comment:
This looks like something done by glualint :thinking: 

Not sure how it got there, and I'm too lazy to git blame it, because it's probably me :joy:

Reply:
Thought so too, it happens :P